storage/storage_src.go

```diff
@@ -93,10 +93,33 @@ func (s *storageImageSource) GetBlob(ctx context.Context, info types.BlobInfo, c
 		return io.NopCloser(bytes.NewReader(image.GzippedEmptyLayer)), int64(len(image.GzippedEmptyLayer)), nil
```

Comment:
Non-blocking: Maybe one more small cleanup: If it makes sense to `.Validate()` the digest at all, conceptually it should be validated before doing the comparison above.

Reply:
Its not strictly needed for the comparison to the empty-layer-digest, but yeah might look nicer. Will do.